packages/odyssey-react-mui/src/Tag.tsx

```diff
@@ -166,13 +190,21 @@ const StyledTag = styled(MuiChip, {
   contrastMode: ContrastMode;
   odysseyDesignTokens: DesignTokens;
   as?: React.ElementType; // Allow the 'as' prop to be forwarded
-}>(({ colorVariant, contrastMode, odysseyDesignTokens }) => {
+  clickable?: boolean;
+}>(({ colorVariant, contrastMode, odysseyDesignTokens, clickable }) => {
   const colors = getChipColors(colorVariant, odysseyDesignTokens, contrastMode);
 
   return {
     backgroundColor: colors.background,
     color: colors.text,
-    borderColor: colors.border,
+
+    ...(clickable === true && {
```

Comment:
Only adding border when these are clickable will cause them to be different sizes. The clickable version will be larger than the non-clickable version

Reply:
ah, I didn't realize they could be used in that context, I updated it to make the border transparent instead.